projects/01-rubygems-cli/solution/errors/bad_request.rb

```diff
@@ -0,0 +1,2 @@
+class BadRequest < StandardError
```

Comment:
Exception classes usually have the "Error" suffix so you can spot them.

Reply:
f993fbfd442105c4af595e6803be0c8ec5142119